dotCMS/src/integration-test/java/com/dotcms/contenttype/test/ContentTypeTest.java

```diff
@@ -0,0 +1,86 @@
+package com.dotcms.contenttype.test;
+
+import static org.junit.Assert.assertEquals;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.Test;
+import com.dotcms.contenttype.model.field.ConstantField;
+import com.dotcms.contenttype.model.field.Field;
+import com.dotcms.contenttype.model.field.ImmutableConstantField;
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotcms.datagen.TestDataUtils;
+import io.vavr.control.Try;
+
+public class ContentTypeTest extends ContentTypeBaseTest {
+
+
+    
+    
+	@Test
+	public void test_that_fields_of_type_are_returned() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5449504)